src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -204,10 +203,10 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
               isSubmitting={isSubmitting}
               register={register}
               watch={watch}
-              {...rest}
-              onSubmit={onFormSubmit}
+              onSubmit={onSubmit}
               editing={!!venueId}
               formError={formError}
+              {...rest}
```

Comment:
You do aware of `rest` overriding all of that, right? Will it cause any problems?

Reply:
I'll check if I can remove it, it seemed unnecessary on first glance.

Reply:
(for context, when it was at the start of the props, the ones set afterwards would override it, but now that it’s moved to the end, it will override all the other props)

Reply:
> (for context, when it was at the start of the props, the ones set afterwards would override it, but now that it’s moved to the end, it will override all the other props)

Yup, it was kinda in the middle, overriding only half of them